### ISSUE TITLE
About cross-validation details

### ISSUE THREAD

**Issue by XueZhangCyy** [lzesyr/DeepMCL#1]
Hello! I would like to know how to design the cross-validation test set and training set in the experiment of this paper. 
For example, for the mHSC_E dataset, what are the TFs assigned to the test and training sets respectively. 
As well as how the results of the final reported AUROC in the paper were obtained, was the average of the AUROC of all test sets for the tri-fold cross-validation taken?
I hope you can answer my questions, thank you very much!
